### DISCUSSION TITLE
What CAPTCHA solvers are available?

### DISCUSSION THREAD

**Discussion by ngregrichardson** [FlareSolverr/FlareSolverr#1047] [Q&A] [answered]
I feel dumb asking this because there is a section in the README.md, but the link to `src/captcha` is a 404. Am I missing some obvious documentation?

**Comment by ilike2burnthing** [comment, marked as answer]
See the warning above it.

**Reply by ngregrichardson** [reply]
🤦 I apparently see gray text and ignore it. Thanks!!
